app/models/application_record.rb

```diff
@@ -0,0 +1,5 @@
+require 'active_record'
```

Comment:
@MothOnMars this is what happens when I remove that line in a demo environment:

```
[search@app1.demo3 current] (deploy)$ RAILS_ENV=production bundle exec rails c
/home/search/i14y/releases/20170704213205/app/models/application_record.rb:1:in `<top (required)>': uninitialized constant ActiveRecord (NameError)
        from /home/search/.bundler/i14y/ruby/2.3.0/gems/activesupport-5.1.1/lib/active_support/dependencies/interlock.rb:12:in `block in loading'
        from /home/search/.bundler/i14y/ruby/2.3.0/gems/activesupport-5.1.1/lib/active_support/concurrency/share_lock.rb:149:in `exclusive'
        from /home/search/.bundler/i14y/ruby/2.3.0/gems/activesupport-5.1.1/lib/active_support/dependencies/interlock.rb:11:in `loading'
        from /home/search/.bundler/i14y/ruby/2.3.0/gems/railties-5.1.1/lib/rails/engine.rb:476:in `block (2 levels) in eager_load!'
        from /home/search/.bundler/i14y/ruby/2.3.0/gems/railties-5.1.1/lib/rails/engine.rb:475:in `each'
        from /home/search/.bundler/i14y/ruby/2.3.0/gems/railties-5.1.1/lib/rails/engine.rb:475:in `block in eager_load!'
        from /home/search/.bundler/i14y/ruby/2.3.0/gems/railties-5.1.1/lib/rails/engine.rb:473:in `each'
        from /home/search/.bundler/i14y/ruby/2.3.0/gems/railties-5.1.1/lib/rails/engine.rb:473:in `eager_load!'
        from /home/search/.bundler/i14y/ruby/2.3.0/gems/railties-5.1.1/lib/rails/engine.rb:354:in `eager_load!'
        from /home/search/.bundler/i14y/ruby/2.3.0/gems/railties-5.1.1/lib/rails/application/finisher.rb:67:in `each'
        from /home/search/.bundler/i14y/ruby/2.3.0/gems/railties-5.1.1/lib/rails/application/finisher.rb:67:in `block in <module:Finisher>'
        from /home/search/.bundler/i14y/ruby/2.3.0/gems/railties-5.1.1/lib/rails/initializable.rb:30:in `instance_exec'
        from /home/search/.bundler/i14y/ruby/2.3.0/gems/railties-5.1.1/lib/rails/initializable.rb:30:in `run'
        from /home/search/.bundler/i14y/ruby/2.3.0/gems/railties-5.1.1/lib/rails/initializable.rb:59:in `block in run_initializers'
        from /usr/local/lib/ruby/2.3.0/tsort.rb:228:in `block in tsort_each'
        from /usr/local/lib/ruby/2.3.0/tsort.rb:350:in `block (2 levels) in each_strongly_connected_component'
        from /usr/local/lib/ruby/2.3.0/tsort.rb:431:in `each_strongly_connected_component_from'
        from /usr/local/lib/ruby/2.3.0/tsort.rb:349:in `block in each_strongly_connected_component'
        from /usr/local/lib/ruby/2.3.0/tsort.rb:347:in `each'
        from /usr/local/lib/ruby/2.3.0/tsort.rb:347:in `call'
        from /usr/local/lib/ruby/2.3.0/tsort.rb:347:in `each_strongly_connected_component'
        from /usr/local/lib/ruby/2.3.0/tsort.rb:226:in `tsort_each'
        from /usr/local/lib/ruby/2.3.0/tsort.rb:205:in `tsort_each'
        from /home/search/.bundler/i14y/ruby/2.3.0/gems/railties-5.1.1/lib/rails/initializable.rb:58:in `run_initializers'
        from /home/search/.bundler/i14y/ruby/2.3.0/gems/railties-5.1.1/lib/rails/application.rb:353:in `initialize!'
        from /home/search/i14y/releases/20170704213205/config/environment.rb:5:in `<top (required)>'
        from /home/search/.bundler/i14y/ruby/2.3.0/gems/railties-5.1.1/lib/rails/application.rb:329:in `require'
        from /home/search/.bundler/i14y/ruby/2.3.0/gems/railties-5.1.1/lib/rails/application.rb:329:in `require_environment!'
        from /home/search/.bundler/i14y/ruby/2.3.0/gems/railties-5.1.1/lib/rails/command/actions.rb:16:in `require_application_and_environment!'
        from /home/search/.bundler/i14y/ruby/2.3.0/gems/railties-5.1.1/lib/rails/commands/console/console_command.rb:84:in `perform'
        from /home/search/.bundler/i14y/ruby/2.3.0/gems/thor-0.19.4/lib/thor/command.rb:27:in `run'
        from /home/search/.bundler/i14y/ruby/2.3.0/gems/thor-0.19.4/lib/thor/invocation.rb:126:in `invoke_command'
        from /home/search/.bundler/i14y/ruby/2.3.0/gems/thor-0.19.4/lib/thor.rb:369:in `dispatch'
        from /home/search/.bundler/i14y/ruby/2.3.0/gems/railties-5.1.1/lib/rails/command/base.rb:63:in `perform'
        from /home/search/.bundler/i14y/ruby/2.3.0/gems/railties-5.1.1/lib/rails/command.rb:44:in `invoke'
        from /home/search/.bundler/i14y/ruby/2.3.0/gems/railties-5.1.1/lib/rails/commands.rb:16:in `<top (required)>'
        from bin/rails:4:in `require'
        from bin/rails:4:in `<main>'
```

Does any of that ring any bells with you?